doc/source/conf.py

```diff
@@ -112,6 +112,26 @@ def setup(app):
 
 html_theme = 'pandas_sphinx_theme'
 
+html_theme_options = {
+    "external_links": [
+        {
+            'url': "https://pandas.pydata.org/pandas-docs/stable/",
+            "name": "Pandas Docs"
+        }
+    ],
+    "github_url": "https://github.com/pandas-dev/pandas-sphinx-theme",
+    "twitter_url": "https://twitter.com/pandas_dev",
```

Comment:
Why did you add these three links to the pandas project? I think the github URL one would be useful but it should point to this repository instead.

Not sure we need a link to twitter, it uses up space we could use for something else.

Reply:
@betatim apologies for the oversight. Will correct this!